src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -47,7 +47,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
```

Comment:
Note: we ONLY have to update the version number for when targeting net452.

For netstandard1.5 we are ALREADY depending on version 4.3.0 (indirectly, so there is no dependency in this file on this package for netstandard1.5).